lib/hello_web/controllers/fallback_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule HelloWeb.FallbackController do
+  @moduledoc """
+  Translates controller action results into valid `Plug.Conn` responses.
+
+  See `Phoenix.Controller.action_fallback/1` for more details.
+  """
+  use HelloWeb, :controller
+
+  # This clause handles errors returned by Ecto's insert/update/delete.
+  def call(conn, {:error, %Ecto.Changeset{} = changeset}) do
+    conn
+    |> put_status(:unprocessable_entity)
+    |> put_view(HelloWeb.ChangesetView)
+    |> render("error.json", changeset: changeset)
+  end
+
+  # This clause is an example of how to handle resources that cannot be found.
+  def call(conn, {:error, :not_found}) do
+    conn
+    |> put_status(:not_found)
+    |> put_view(HelloWeb.ErrorView)
```

Comment:
@keliumJU what does `put_view` do?

Reply:
put_view prepares a view to be rendered, we can choose any of our available views with put_view and then with the pipe operator |> pass the necessary parameters